components/resources/wallet_strings.grdp

```diff
@@ -1145,6 +1145,11 @@
   <message name="IDS_BRAVE_WALLET_CLEAR_FILTERS" desc="Clear filters label">Clear filters</message>
   <message name="IDS_BRAVE_WALLET_SHOW_MORE" desc="Show more label">Show more</message>
   <message name="IDS_BRAVE_WALLET_DETAILS" desc="Details label">Details</message>
+  <message name="IDS_BRAVE_WALLET_SWITCH_TO_SHIELDED_ACCOUNT" desc="Switch to shielded account button text">Switch to shielded account</message>
+  <message name="IDS_BRAVE_WALLET_SHIELD_ACCOUNT" desc="Shield account button text">Shield account</message>
+  <message name="IDS_BRAVE_WALLET_ACCOUNT_NOT_SHIELDED_DESCRIPTION" desc="ZCash account not shielded description.">Currently this account supports transparent transactions which means they are visible to everyone on the blockchain.</message>
+  <message name="IDS_BRAVE_WALLET_ACCOUNT_SHIELDED_DESCRIPTION" desc="ZCash account shielded description">Upgrading to shielded account means that these transactions hide the sender, receiver and amount details.</message>
```

Comment:
Should it be "Upgrading to **a* shielded account means..."?

Reply:
Yes that makes more sense, updated!

![Screenshot 38](https://github.com/user-attachments/assets/aee4ced6-d955-4035-b3f5-28da384f5804)